### ISSUE TITLE
docker-entrypoint.sh ignore arguments

### ISSUE THREAD

**Issue by dsuievalov** [newrelic/newrelic-php-daemon-docker#19]
### Description
docker-entrypoint-template ignore arguments because in POSIX sh string indexing is undefined (https://github.com/koalaman/shellcheck/wiki/SC2039)

Current behavior

First if condition will never be triggered in POSIX sh:

```
if [ "${1:0:1}" = '-' ]; then
  set -- /usr/bin/newrelic-daemon $defaultArgs "$@"
elif [ "$1" = '/usr/bin/newrelic-daemon' ]; then
  # Remove the first element from the arguments
  shift 1
  set -- /usr/bin/newrelic-daemon $defaultArgs "$@"
fi
```
Example:
```
/ # a="--1"
/ # echo "${1:0:1}"

/ #
```

Expected behavior

If case for arguments should be triggered.

**Comment by zsistla** [comment]
Hi Dsuievalov,

Thank you for noticing the POSIX incompatibility. We will file an internal issue for this and keep you updated about the progress.

Thanks for using the the New Relic PHP Agent.

**Comment by zsistla** [comment]
Hi Dsuievalov,

The POSIX incompatibility issue has been addressed in commit: [7ae2c802].

The change will take effect in our next release.

Thanks again for you feedback and for using the PHP Agent.
